administration/src/graphql/applications/getApplication.graphql

```diff
@@ -0,0 +1,14 @@
+query getApplication($applicationVerificationId: Int!, $accessKey: String!) {
```

Comment:
Maybe `getApplicationVerification` would be a better naming, since we need a `getApplication` query for the user

Reply:
Naming was adjusted to `getApplicationByApplicationVerificationAccessKey`